REFERENCE.md

```diff
@@ -649,6 +654,46 @@ Specify upstream (Ubuntu) PPA entry.
 
 Default value: `undef`
 
+##### <a name="-redis--apt_location"></a>`apt_location`
+
+Data type: `Optional[String]`
+
+Specify the URL of the apt repository.
+
+Default value: `https://packages.redis.io/deb/`
```

Comment:
This doesn't look like the real default. Normally we generate the reference using `rake strings:generate:reference`